Panic on startup: nil pointer dereference in BlackholeHandler before Queue is initialized

## Description
Shortly after starting the daemon, the web server can panic with a nil pointer dereference when a request hits `/blackhole` before `DirectoryWatcherService.Queue` has been initialized.

## Steps to reproduce
1. Start `premiumizearrd` with the web UI enabled.
2. Open (or already have open) the dashboard in a browser right at startup, so it polls `/blackhole` immediately.
3. If resolving the premiumize.me folder (`GetDownloadsFolderIDFromPremiumizeme`) takes any noticeable time, a request can land while `Queue` is still nil.

## Stack trace
panic: runtime error: invalid memory address or nil pointer dereference
...
github.com/ensingerphilipp/premiumizearr-nova/pkg/stringqueue.(*StringQueue).GetQueue(...)
pkg/stringqueue/stringqueue.go:33
github.com/ensingerphilipp/premiumizearr-nova/internal/service.(*WebServerService).BlackholeHandler(...)
internal/service/web_service_handlers.go:97



## Root cause
In `DirectoryWatcherService.Start()`, `dw.Queue` is only created *after* the premiumize.me API call that resolves `downloadsFolderID`:

--- a/internal/service/directory_watcher_service.go
+++ b/internal/service/directory_watcher_service.go
@@ -72,10 +72,10 @@ func (dw *DirectoryWatcherService) GetStatus() string {
 func (dw *DirectoryWatcherService) Start() {
 	log.Info("Starting directory watcher...")
 
-	dw.downloadsFolderID = utils.GetDownloadsFolderIDFromPremiumizeme(dw.premiumizemeClient, dw.config.TransferDirectory)
-
 	log.Info("Creating Queue...")
 	dw.Queue = stringqueue.NewStringQueue()
+
+	dw.downloadsFolderID = utils.GetDownloadsFolderIDFromPremiumizeme(dw.premiumizemeClient, dw.config.TransferDirectory)
 
 	log.Info("Starting uploads processor...")
 	go dw.processUploads()

--- a/internal/service/web_service_handlers.go
+++ b/internal/service/web_service_handlers.go
@@ -91,7 +91,7 @@ func (s *WebServerService) DownloadsHandler(w http.ResponseWriter, r *http.Reque
 func (s *WebServerService) BlackholeHandler(w http.ResponseWriter, r *http.Request) {
 	var resp BlackholeResponse
 
-	if s.directoryWatcherService == nil {
+	if s.directoryWatcherService == nil || s.directoryWatcherService.Queue == nil {
 		resp.Status = "Not Initialized"
 	} else {
 		for i, n := range s.directoryWatcherService.Queue.GetQueue() {

